batch_2/aarti_exercise/lesson01.js

```diff
@@ -0,0 +1,2 @@
+var a = 8;
+console.log(A);
```

Comment:
'A' will raise error saying undecalred variable/undefined variabl. Please use 'a'.

Reply:
will change it